libsolidity/formal/SMTEncoder.cpp

```diff
@@ -560,6 +560,23 @@ bool SMTEncoder::visit(Conditional const& _op)
 	return false;
 }
 
+bool SMTEncoder::visit(FunctionCall const& _funCall)
+{
+	auto functionCallKind = *_funCall.annotation().kind;
+	if (functionCallKind != FunctionCallKind::FunctionCall)
+		return true;
+
+	FunctionType const& funType = dynamic_cast<FunctionType const&>(*_funCall.expression().annotation().type);
+	// We do not want to visit the TypeTypes in the second argument of `abi.decode`.
```

Comment:
If it is visited/inspected somewhere else, please leave a not to that effect here.

Reply:
Done